packages/run-protocol/src/vaultFactory/liquidateMinimum.js

```diff
@@ -85,7 +93,11 @@ const start = async zcf => {
         });
       };
       await sellAllIfUnsold();
-
+      const proceeds = debtorSeat.getAmountAllocated('RUN', runBrand);
+      trace(`Liq results`, {
+        runDebt,
+        proceeds,
+      });
```

Comment:
the `const proceeds` reads at first like it might be part of the logic. to isolate consider,
```suggestion
      const proceeds = debtorSeat.getAmountAllocated('RUN', runBrand);
      trace(`Liq results`, {
        runDebt,
        debtorSeat.getAmountAllocated('RUN', runBrand),
      });
```

Reply:
DRY!  Also, what I want to trace is the value that would be used, so I never want to do additional retrieval work in a `trace` statement.

Reply:
oh yes. This also gives it a name in the trace result.

Reply:
I mistyped. What I'm suggesting is:
```suggestion
      trace(`Liq results`, {
        runDebt,
        proceeds: debtorSeat.getAmountAllocated('RUN', runBrand),
      });
```
That's assuming `proceeds` will only be used for diagnostics.